config/samples/canary.yaml

```diff
@@ -84,10 +84,16 @@ spec:
           app: appinfo
```

Comment:
I'm not sure about all the `/samples/*` files. Why do we have them here anyway? Kafka, Zookeeper, Mysql and Flink are already part of the operators repo. The rest is more like snippets. Is it needed or is it a remnant? Should we just remove the `samples` folder? /cc @gerred @fabianbaier

Reply:
Remnant. Nuke it. :)